src/components/modals/ScheduleModal.jsx

```diff
@@ -25,8 +25,8 @@ const ScheduleModal = ({
       <label htmlFor="name">Name*</label>
       <input id="name" type="text" placeholder="Name" {...register("name")} className="modalFields p-2" />
       */}
-      <label htmlFor="start_dt">Start DateTime</label>
-      <input id="start_dt" type="datetime-local" placeholder="Start DateTime" {...register("start_dt", {})} className="modalFields p-2" />
+      <label htmlFor="start_datetime">Start DateTime</label>
```

Comment:
@prykon this also breaks backward compatibility bc the form id fields get stored as json in the DB table records, and older ones will now not set the value on the form when loaded.  I think this "dt" -> "datetime" is a little too specific. I wouldn't have a problem if they were new fields, but since they are existing, we should leave as-is

Reply:
Rolled back.